core/jazz_ui/protractor.conf.js

```diff
@@ -52,12 +52,8 @@ exports.config = {
     });
     jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
     jasmine.getEnv().addReporter(new Jasmine2HtmlReporter({savePath: 'target/screenshots'}));
-    jasmine.getEnv().addReporter(new JSONReporter({
-      file: 'jasmine-test-results.json',
-      beautify: true,
-      indentationLevel: 4 
-    }));
-    browser.manage().timeouts().implicitlyWait(2000000);
+    //browser.manage().timeouts().implicitlyWait(2000000);
+    browser.manage().timeouts().implicitlyWait(600000);
```

Comment:
as we are explicitly waiting in code. Can we remove this implicit wait?

Reply:
We have to give the implicit as this is require in test suit. because every page need some time to upload their elements. I reduced the implicit time to 6 min.

Reply:
Json reporter is used for creating the Test Summary.Can this be reverted?